src/shared/codelens/localLambdaRunner.ts

```diff
@@ -357,59 +362,87 @@ export const makeBuildDir = async (): Promise<string> => {
 
 export async function makeInputTemplate(params: {
     baseBuildDir: string,
-    codeDir: string,
+    templateDir: string,
     documentUri: vscode.Uri
     originalHandlerName: string,
     handlerName: string,
     runtime: string,
     workspaceUri: vscode.Uri,
-}): Promise<string> {
+    channelLogger: ChannelLogger
+}): Promise<InputTemplateInfo> {
+
+    // Switch over to the output channel so the user has feedback that we're getting things ready
+    params.channelLogger.channel.show(true)
+
+    params.channelLogger.info(
+        'AWS.output.sam.local.start',
+        'Preparing to run {0} locally...',
+        params.handlerName
+    )
+
     const inputTemplatePath: string = path.join(params.baseBuildDir, 'input', 'input-template.yaml')
     ExtensionDisposableFiles.getInstance().addFolder(inputTemplatePath)
 
     // Make function handler relative to baseDir
     const handlerFileRelativePath = path.relative(
-        params.codeDir,
+        params.templateDir,
         path.dirname(params.documentUri.fsPath)
     )
 
     const workspaceFolder = vscode.workspace.getWorkspaceFolder(params.workspaceUri)
+    let existingLambda: LocalLambda | undefined
     let existingTemplateResource: CloudFormation.Resource | undefined
     if (workspaceFolder) {
 
-        const relativeOriginalFunctionHandler = normalizeSeparator(
-            path.join(
-                handlerFileRelativePath,
-                params.originalHandlerName,
+        let relativeOriginalFunctionHandler = params.originalHandlerName
+        if (!params.runtime.includes('dotnet')) {
```

Comment:
See `getRuntimeFamily` from `src/lambda/models/samLambdaRuntime.ts` (same comment throughout this file).

Reply:
✔️